plugins/scenario/Runner.ts

```diff
@@ -74,7 +91,7 @@ export class Runner<T, U, R> {
     let cumulativeGas = 0;
     for (const combo of combos(baseSolutions.concat(solutionChoices))) {
       // create a fresh copy of context that solutions can modify
-      let ctx: T = await scenario.forker(context, world);
+      let ctx: T = await env.initializer(world);
```

Comment:
Seems like forking context wasn't actually needed then?

Reply:
Oh I was wondering where this went, I actually had some old changes that I think actually slipped in removing forking first, I should maybe add it back, its possibly a little weird conceptually right now. I'll take another look at this tmr

Reply:
Added a note about it, its not really necessary at all right now, but could be added back if we need to communicate something from static constraints to dynamic ones